src/orders-v2/order-header.tsx

```diff
@@ -0,0 +1,85 @@
+import { channelUrl } from "@dashboard/channels/urls";
+import { TopNav } from "@dashboard/components/AppLayout";
+import { OrderStatus } from "@dashboard/graphql";
+import { transformOrderStatus } from "@dashboard/misc";
+import { Box, Button, Text } from "@saleor/macaw-ui-next";
+import { Package } from "lucide-react";
+import moment from "moment-timezone";
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { StatusPill } from "./status-pill";
+import { UnderlineLink } from "./underline-link";
+
+interface Props {
```

Comment:
let's ensure all components extends `BoxProps`, it's a basic composition that allows us to position components by parents